core/src/main/java/com/sequenceiq/cloudbreak/service/StackCommonService.java

```diff
@@ -78,6 +79,9 @@
     @Inject
     private StackCreatorService stackCreatorService;
 
+    @Autowired
```

Comment:
We use @Inject, please change the annotation

Reply:
Ok, I will do that.